docs/wiki/api-references/Statistical-Nodes-API.md

```diff
@@ -2,7 +2,7 @@ This page contains the documentation for the `csp.stats`  library. The
 `stats`  library contains functions to calculate statistics on time
 series data over rolling windows.
 
-# API Reference
+## Table of Contents
 
 1. **Base Statistics**
 
```

Comment:
Do we need to keep the numbering at the second level here? I'm not sure this is very useful and might be confusing, maybe just bullet points would be best.

Reply:
Agreed, and updated :)